eos-phone-home

```diff
@@ -60,9 +60,15 @@ import requests
 
 log = logging.getLogger(__name__)
 
+Config = collections.namedtuple(
+    'Config',
+    ['host', 'debug', 'force', 'exit_on_server_error'],
+)
```

Comment:
I guess it makes sense to go with the flow of the surrounding code, but I think today I would use:

```python
from dataclasses import dataclass
...
@dataclass
class Config
    host: str
    debug: bool
    force: bool
    exit_on_server_error: bool
```

Reply:
Indeed, I meant to leave a comment about that. dataclass is a better fit and I'd be happy to rework. I'll give it a shot.

Reply:
That looks nicer and the defaults are more properly stored in the `Config` class.